modules/monitoring-listing-alerting-rules.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-your-own-services.adoc
+
+[id="listing-alerting-rules_{context}"]
+= Listing alerting rules
+
+You can list existing alerting rules for a project.
```

Comment:
Do we need to mention `oc get -n <namespace> prometheusrule blah -oyaml`? As just seeing the list of them, is not that useful, actually viewing what is part of it might be helpful as well?